src/api/operations/staffAssignment.fragments.graphql

```diff
@@ -25,11 +16,9 @@ fragment StaffAssignmentDetails on StaffAssignment {
   staffAssignmentRelationship
   assignedAt
   unassignedAt
-  household {
```

Comment:
I was wondering why I even included this originally, since `StaffAssignmentDetails` is only ever resolved on a household or enrollment, where we already have the household ID. I thought maybe it was related to graphql caching. (see https://github.com/greenriver/hmis-frontend/pull/856#discussion_r1693010151?) But I clicked around, including assigning and unassigning staff, and wasn't able to create any weird behavior, so I think it makes sense to remove.

Reply:
Ah okay, thanks for poking around at that. I agree I don't see that it's necessary. I removed it as a performance improvement.